x/oracle/simulation/genesis.go

```diff
@@ -21,13 +22,13 @@ func PortFn(r *rand.Rand) string {
 	if r.Intn(2) > 0 {
 		return "oracle"
 	}
-	length := uint64(randIntBetween(r, 6, 10))
+	length := uint64(helpers.RandIntBetween(r, 6, 10))
```

Comment:
I'm not sure why you chose to comment about error handling on this line since there aren't any errors to handle here.

I did update `OracleFn`, though, to safely handle the case when `SelectRandomAccounts` doesn't return any accounts.